openedx.yaml

```diff
@@ -1,13 +1,16 @@
 # This file describes this Open edX repo, as described in OEP-2:
 # http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
 
-owner: tuchfarber
```

Comment:
I thought we should pin the `owner` to a team instead of a person. How about `Masters and Micromasters Squad`?

Reply:
Or simply `masters-devs`

Reply:
`masters-devs` sounds good